docs/SRS/SRS.tex

```diff
@@ -1310,7 +1311,7 @@ \subsection{Reflection}
 	      \dots
 
 	      \textbf{Joy Xiao}
-
+	      One skill I would like to improve is team management because this is a large project which we have to manage by ourselves and pace ourselves for the whole school year.
 	      \dots
```

Comment:
```suggestion

	      One skill I would like to improve is team management because this is a large project which we have to manage by ourselves and pace ourselves for the whole school year.
```